package io.ylab.monitoring.domain.auth.exception;

public class UserNotFoundException extends AuthMonitoringException {
}
